route/middleware.go

```diff
@@ -17,6 +17,28 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func (r *Router) queryTokenChecker(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		requiredToken := r.Config.GetQueryAuthToken()
+		token := req.Header.Get(types.QueryTokenHeader)
+		fmt.Printf("required: %s  specified: %s\n", requiredToken, token)
```

Comment:
Should we use logrus here instead?

Reply:
Nah, we should delete the printf I left in by mistake. :)